parm/config/config.resources

```diff
@@ -195,6 +195,16 @@ elif [ $step = "post" ]; then
     export nth_post=1
     export npe_node_post=12
     export npe_node_dwn=$npe_node_max
+    if [ $machine = "WCOSS_DELL_P3" ]; then
+      export npe_dwn=28
+    else
+      export npe_dwn=24
+    fi
+    if [ "$WRITE_DOPOST" = ".true." ]; then 
+      export npe_post=$npe_dwn
```

Comment:
This is significantly different than the one on line 194.
Can you explain in short what is `WRITE_DOPOST` intended use?

Reply:
The WRITE_DOPOST is a switch for inline-post. When it is turned on, the post job use much less system resource. This modification will allow post job to use less resource and get to run faster in supercomputer.

Reply:
Answered in File Changed page.